shell/machine-config/vmwarevsphere.vue

```diff
@@ -417,12 +418,18 @@ export default {
 
       const url = `/meta/vsphere/${ resource }?${ queryParams }`;
 
-      const result = await this.$store.dispatch('management/request', {
-        url,
-        redirectUnauthorized: false,
-      }, { root: true });
+      try {
```

Comment:
I wanted to more gracefully handle and report request issues to the console. I just wrapped this in a try catch and log the errors.

This came about because the account I'm using does seem to have access to content libraries.